### PR TITLE
Consistency of codes + website

### PR DESCRIPTION
**Pull request by samwaseda**
In response to the [major update issue](https://github.com/pyiron/pyiron/issues/773), I started to look at the consistency between codes and the content on the website. Feel free to add more stuff.

### CODE REVIEW

**Comment by samwaseda** [comment]
@jan-janssen Please add this to the list of topics for this Friday.

File: pyiron/lammps/control.py [hunk 1]
```diff
@@ -250,8 +250,10 @@ def pressure_to_lammps(self, pressure, rotation_matrix):
 
     def calc_minimize(
```
Comment (jan-janssen) [inline pyiron/lammps/control.py:251]: This function is never called from the user directly, so we do not need `e_tol` and `f_tol` here.

File: pyiron/lammps/base.py [hunk 2]
```diff
@@ -573,25 +573,41 @@ def collect_output_log(self, file_name="log.lammps", cwd=None):
...
+            ionic_energy_tolerance = e_tol
+            e_tol = None
+        if f_tol is not None:
+            warnings.warn(
+                "f_tol is deprecated as of vers. 0.3.0. It is not guaranteed to be in service in vers. 0.4.0. Use ionic_force_tolerance instead.",
+                DeprecationWarning
+            )
+            ionic_force_tolerance = f_tol
+            f_tol = None
         super(LammpsBase, self).calc_minimize(
```
Comment (jan-janssen) [inline pyiron/lammps/base.py:601]: Here we should set both - e_tol, f_tol and ionic_energy_tolerance, ionic_force_tolerance so the warnings are raised from AtomisticGenericJob.

File: pyiron/lammps/control.py (hunk 1 above)
Comment (samwaseda) [reply pyiron/lammps/control.py:251]: Shouldn’t it be a private function then? In general I was not really sure which functions can be used by the user and which ones cannot be.

Comment (jan-janssen) [reply pyiron/lammps/control.py:251]: The issue is that pep8 defines private functions as those which should only be used within the same class and derived. But in pyiron we have three levels: 

- user level - what is used in the notebook
- developer level - what is used in derived classes but should never be called from the user directly
- class level - what is only used within one class but not in other classes - _private functions_

**Comment by jan-janssen** [comment]
@samwaseda Can you merge this one?

**Pull request merged**